timer/timer.go

```diff
@@ -1,7 +1,11 @@
 package timer
 
+import "time"
+
 // Timer is a metric time tracking interface
 type Timer interface {
+	// StartAt starts timer at a given time
+	StartAt(time.Time) Timer
```

Comment:
please move this method a bit down - `Start()` is more important for this interface

Reply:
sure =)